assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -249,6 +313,47 @@ def _update_conda_dependencies(self, package_details):
             yaml.safe_dump(conda_dict, f)
             logger.info("updated conda.yaml")
 
+    def _get_curated_environment_pip_package_list(self) -> List[str]:
+        """
+        Retrieve the packages using 'conda list' command.
+
+        :return: A List of the pip package and the corresponding versions.
+        """
+        import subprocess
+        import json
+
+        PIP_LIST = ['accelerate', 'cffi', 'dill', 'google-api-core', 'numpy',
+                    'packaging', 'pillow', 'protobuf', 'pyyaml', 'requests', 'scikit-learn',
+                    'scipy', 'sentencepiece', 'torch', 'mlflow']
+        ADD_PACKAGE_LIST = ['torchvision==0.14.1', 'transformers==4.34.1']
```

Comment:
why hardcode?

Reply:
we would want to make use of model-mgmt env to get he dependencies in all case.

Reply:
Moved the pkgs to the conda yaml for whisper, the others we had a conversion on the fact that the version is being taken from the latest env